app/init.php

```diff
@@ -610,9 +610,17 @@ function (mixed $value, Document $document, Database $database) {
             ])
         ));
         if (\count($targetIds) > 0) {
-            return $database->find('targets', [
-                Query::equal('$internalId', $targetIds)
-            ]);
+            try {
+                $database->disableValidation();
+                $targets = $database->find('targets', [
+                    Query::equal('$internalId', $targetIds)
+                ]);
+                return $targets;
+            } catch (\Throwable $th) {
+                return [];
+            } finally {
+                $database->enableValidation();
+            }
```

Comment:
I don't think we should try/catch here, it will be hard to debug if something goes wrong

Reply:
Updated